### PR TITLE
WIP feat: User Login Page

### LINKED ISSUE #231: Implement User Login
**Problem**
Create a user login page at /login, per the screenshot below.

**Solution**
Implement according to screenshot/wireframe below. Note: Federated login (e.g. Facebook/ Google, etc) will be implemented in later milestones, e.g. 3.2 or later.
Please also add a "Request Access" link, and you can delete the Organization "Login" button, since only users login.

**Screenshots**
https://projects.invisionapp.com/share/TMVDTBZ7PE3#/screens/309925229_Login
User Login

### PR DESCRIPTION
**Pull request by BradenM**
**Changes**

* Implementing user login page as outlined in #231 
* Added typography variants to tailwind

Closes #231

### CODE REVIEW

File: src/pages/Login.vue [hunk 1]
```diff
@@ -60,37 +78,62 @@ export default {
...
 
 <style scoped>
-.container {
+.grid-container {
   display: grid;
-  grid-template-columns: 40px 1fr 1fr 1fr 1.5fr;
-  grid-template-rows: 1fr 2fr 1fr;
+  grid-template-columns: repeat(6, 1fr);
+  grid-template-rows: 20% auto;
+  grid-template-areas:
```
Comment (tabiodun) [inline src/pages/Login.vue]: Love it

File: src/layouts/Unauthenticated.vue [hunk 2]
```diff
@@ -11,4 +11,20 @@ export default {
 };
 </script>
 
-<style></style>
+<style>
+html,
+body,
+.wrapper,
+#app {
+  height: 100%;
```
Comment (tabiodun) [inline src/layouts/Unauthenticated.vue]: can you use `@apply w-full h-full` here?

Comment (BradenM) [reply src/layouts/Unauthenticated.vue]: Definitely. As you can probably tell, not quite as confident yet in Grid as I am flex haha. Got a solid grasp of it now, but also just noticed that [tailwind has an entire section dedicated to grid helpers](https://tailwindcss.com/docs/grid-template-columns)... so probably will be redoing some of this to utilize those.

Comment (tabiodun) [reply src/layouts/Unauthenticated.vue]: Actually I think their grid is flex based. I think the way you're doing the grid is good. I'm just saying you can use the apply to use the tailwind height/width helpers

Comment (BradenM) [reply src/layouts/Unauthenticated.vue]: Oh really? Had not looked into it much yet. In that case sounds good then.

Working on getting the basic phone API merged today, then will try and finish this up.

**Pull request merged**